Products/ZenUI3/browser/device_export.py

```diff
@@ -13,10 +13,24 @@
 from Products.ZenUtils.jsonutils import unjson
 from Products.Zuul.facades.devicefacade import DeviceFacade
 from xml.etree.ElementTree import Element, tostring
+from Products.ZenEvents.EventClass import EventClass
 
 log = logging.getLogger('zen.deviceexporter')
 
 
+def event(values):
+    # Get event's types with non-empty count field
+    values = {x:y.get('count') for x,y in values.items() if y.get('count')}
```

Comment:
Yes, I do not need values where "count" is 0. Will change "non-empty" to "non-zero" in comment.